tool/llpcAutoLayout.cpp

```diff
@@ -53,6 +53,12 @@ using namespace llvm;
 using namespace Llpc;
 using namespace SPIRV;
 
+struct ResourceNodeSet
```

Comment:
Add the comment for the whole structure

// Represent the info of a collection of resource nodes, including their binding maps.
strcuct ResourceNodeSet